### ISSUE TITLE
Rillaboom missing?

### ISSUE THREAD

**Issue by Meister-anon** [rh-hideout/pokeemerald-expansion#1439]
Looking through the repository I don't see any graphic information for Rillaboom at all.
The pre-evolutions and most other gen 8 pokemon are there. Was it forgotten?

**Comment by LOuroboros** [comment]
> 
> 
> Looking through the repository I don't see any graphic information for Rillaboom at all.
> The pre-evolutions and most other gen 8 pokemon are there. Was it forgotten?

I don't think it was, as its sprite data in `src/data/graphics/pokemon.h` seems to be commented out on purpose, just like Intelleon's.

If I had to guess, I'd say their sprites weren't available in the public repository of sprites that we've used for the majority of the species sprite work in the PE branch. They *are* available now, so it'll be addressed eventually.

Maybe I'll submit a PR myself later today since it's pretty quick to sort out.
